src/cdomains/intDomain.ml

```diff
@@ -2589,7 +2637,7 @@ module IntDomTupleImpl = struct
     ); !dt
 
   let no_overflow ik r =
-    if GobConfig.get_bool "ana.int.congruence_no_overflow" && Cil.isSigned ik then true
+    if Cil.isSigned ik  && GobConfig.get_string "sem.int.signed_overflow" = "assume_none" then true
```

Comment:
This and the two following changes could just be `should_ignore_overflow` calls.